### PR TITLE
Adding unit tests for `fixed_point` with extremely large `scale`s

### PR DESCRIPTION
**Pull request by codereport**
After a discussion with Keith and Ashwin today, I realized `libcudf` was missing a couple corner cases for `fixed_point` and decided to open a small PR to add them.

### CODE REVIEW

**Review by vuule — APPROVED** [review]

File: cpp/tests/fixed_point/fixed_point_tests.cu
Comment (kkraus14) [inline cpp/tests/fixed_point/fixed_point_tests.cu:607]: Should we similarly add large negative scale tests as well?

Comment (codereport) [reply cpp/tests/fixed_point/fixed_point_tests.cu]: Pushed :)

**Review by kkraus14 — APPROVED** [review]

**Comment by codereport** [comment]
@gpucibot merge

**Comment by harrism** [comment]
@codereport we require two approving C++ codeowner reviews before merging.

**Review by harrism — CHANGES_REQUESTED** [review]
Can you please add documentation of what the corner cases are that you are testing? I don't quite understand.

**Comment by codereport** [comment]
> @codereport we require two approving C++ codeowner reviews before merging.

Sorry, I thought @kkraus14 qualified as a C++ reviewer for this PR because it was pretty straight forward.

**Review by hyperbolic2346 — APPROVED** [review]

**Review by nvdbaranec — APPROVED** [review]

**Review by harrism — APPROVED** [review]
Thanks for documenting.

**Comment by harrism** [comment]
@gpucibot merge

**Pull request merged**
